docs/feature_velocikey.md

```diff
@@ -16,15 +17,43 @@ VELOCIKEY_ENABLE = yes
 Then, while using your keyboard, you need to also turn it on with the VLK_TOG keycode, which toggles the feature on and off.
 
 The following light effects will all be controlled by Velocikey when it is enabled:
- - RGB Breathing
- - RGB Rainbow Mood
- - RGB Rainbow Swirl
- - RGB Snake
- - RGB Knight
 
-Support for LED breathing effects is planned but not available yet.
+-   RGB Breathing
+-   RGB Rainbow Mood
+-   RGB Rainbow Swirl
+-   RGB Snake
+-   RGB Knight
 
- As long as Velocikey is enabled, it will control the speed regardless of any other speed setting that your RGB lights are currently on.
+Support for LED breathing effects is planned but not available yet.
```

Comment:
Do you mean by this LED Backlight?

Reply:
I think this is a glitch by `diff`, because all I did was remove line 30, as I added the configurability via `config.h`, as documented below.